services/forwarding/forwarding.rst

```diff
@@ -0,0 +1,90 @@
+.. _doc_mailforwarding:
+.. _Dashboard: https://mailsac.com/dashboard
+.. _`Manage Email Addresses`: http://mailsac.com/addresses
+.. _`Manage Domains`: http://mailsac.com/domains
+
+Email Forwarding
+================
+
+Forwarding allows email to be sent to a different email address or
+service (Webhook, Websocket, Slack Webhook). Private addresses and custom
+domains can be configured for email forwarding. Forwarding is not available on
+disposable email addresss.
+
+Forward to Another Mailsac Address
+----------------------------------
+
+Any private address can be forwarded to another private address.
+This allows you to consolidate many email addresses into a single inbox.
+
+Forwarding to another private address can be configured by selecting
+`Manage Email Addresses`_ from the Dashboard_. Select the *Settings* button
+next to the email address to manage, then choose the private address to forward
+to from the *Forward to Another Email Address* dropdown and select *Save
+Settings* 
+
+For example, if you own 1@mailsac.com, 2@mailsac.com, 3@mailsac.com, and
+main@mailsac.com, you could setup the following scheme:
+
+* 1@mailsac.com forwards to main@mailsac.com
+* 2@mailsac.com forwards to main@mailsac.com
+* 3@mailsac.com forwards to main@mailsac.com
+* main@mailsac.com is checked by POP3 in GMail
+
+Catch-All Domain Forwarding Addresses
+-------------------------------------
+
+A :ref:`Catch-All email address <doc_catchall>` can receive all the mail for
+a custom domain, and optionally forward it to another address or service (
+Webhook, Websocket, or Slack Webhook). A Catch-All address is a private address
+in the format `*@example.com`.
+
+A Catch-All email address can be configured by selecting `Manage Domains`_ from
+the Dashboard_, then choosing *Manage* next to the domain, and then selecting
+the *Forwarding* tab.
+
+Slack Email Forwarding
+----------------------
+
```

Comment:
Another note might be useful:

- To forward all inbound email to a custom domain, configure a Catch-All inbox on the domain, then add Slack forwarding to the Catch-All inbox. This allows sending many inbound private email addresses to slack, without confinguring each one individually.